frontend/vite.config.ts

```diff
@@ -18,7 +19,41 @@ const config = ({ mode }: ConfigEnv): ReturnType<typeof defineConfig> => {
     build: {
       outDir: 'build',
     },
-    plugins: [tsconfigPathsPlugin(), reactPlugin(), svgr(), pluginRewriteAll()],
+    plugins: [
+      tsconfigPathsPlugin(),
+      reactPlugin(),
+      svgr(),
+      pluginRewriteAll(),
+      VitePWA({
+        registerType: 'autoUpdate',
+        includeAssets: ['favicon.ico', 'favicon.svg', 'apple-touch-icon.png'],
+        manifest: {
+          name: 'Writorium',
+          short_name: 'Writorium',
+          description: 'Unbounded space for freedom of your feather',
+          theme_color: '#2e453b',
+          start_url: '/',
+          icons: [
+            {
+              src: '/android-chrome-192x192.png',
+              sizes: '192x192',
+              type: 'image/png',
+            },
+            {
+              src: '/android-chrome-512x512.png',
+              sizes: '512x512',
+              type: 'image/png',
+            },
+            {
+              src: '/maskable_icon_x512.png',
+              sizes: '512x512',
+              type: 'image/png',
+              purpose: 'maskable',
+            },
+          ],
+        },
+      }),
```

Comment:
Can we move it to a constant?

Reply:
made in the same file:

[79536b8](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/425/commits/79536b8cde4072cc8832e85f1272c2f377aa462d)